### PR TITLE
C make files

### PR DESCRIPTION
**Pull request by carlyrhoover**
### Fixes #29 

---

### **What was changed?**
-CMakeLists.txt was modified. Include files were edited in plugin. 
---

### **Why was it changed?**

Platform was not able to be defined, so it was specified in the CmakeLists.txt. Header file was also not being properly included, so it was removed to avoid that error. The project builds properly, but is unable to be tested due to the nature of the plugin itself, which is an issue to be addressed in the future. 

---

### **How was it changed?**

No major changes. Includes in the plugin file were deleted, and CMakeLists.txt was edited.

### CODE REVIEW

**Comment by harshitha-thota** [comment]
Suggested Improvements:
Comment out the test stuff from CMakeLists.txt(don’t remove, just comment out).

Please address the above suggested improvement. After testing, I’ll merge the PR if everything works as expected. I look forward to seeing your updates!

File: src/hello-world.cpp
Comment (kate-holdener) [inline src/hello-world.cpp:1]: Remove this from the PR, I added this file just to see if we can build it with cmake.

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+ # Include the 'include' directory for plugin_tests
+ target_include_directories(plugin_tests PRIVATE
+    ${XPLANE_SDK_PATH}/CHeaders/XPLM
+    ${XPLANE_SDK_PATH}/CHeaders/Widgets
+    include
+)
+
+include(GoogleTest)
+gtest_discover_tests(plugin_tests)
```
Comment (kate-holdener) [inline CMakeLists.txt:76]: Comment out this part, since it doesn't quite work with the plugin_test.cpp yet.

**Review by kate-holdener — CHANGES_REQUESTED** [review]
@carlyrhoover please address the requested changes.

**Comment by carlyrhoover** [comment]
The branch is now updated with the testing code in the CMakeFiles commented out!

**Review by harshitha-thota — APPROVED** [review]

**Pull request merged**